### PR TITLE
socket.io working basic

### PR DESCRIPTION
**Pull request by aribengiyat**
Added basic functionality for socket.io on the frontend and backend. I used the documentation to understand how events are emitted and listened for. You can emit custom events and listen for them appropriately in the front end.

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
   <title>Test Page</title>
 </head>
 <body>
-  <h1>The best chat app number 1 the best!!!!!!</h1>
+  <h1>Testing web sockets</h1>
+  </div>
```
Comment (aribengiyat) [inline client/public/index.html]: hanging div tab -- remove

File: client/src/index.js [hunk 2]
```diff
@@ -1 +1,15 @@
...
+  console.log('From server banana: ', message);
+  document.querySelector('.chat').innerHTML = message;
+})
+
+
+const sendMessage = () => {
+  socket.emit('frontendMessage', '{username: test}')
+}
+
+window.sendMessage = sendMessage;
```
Comment (aribengiyat) [inline client/src/index.js]: should be removed polutes global namespace

**Review by aribengiyat — COMMENTED** [review]
Some changes commented by team

**Review by GarrettYan — APPROVED** [review]
Review done, everything is ok.

**Pull request merged**